### PR TITLE
RAIN-2442 Added safe html annotation in remaining module of municipal…

### PR DESCRIPTION
**Pull request by rohit-eGov**
… service

### CODE REVIEW

File: property-services/src/main/java/org/egov/pt/consumer/FileStoreConsumer.java [hunk 1]
```diff
@@ -97,7 +97,7 @@ private Property searchProperty(String id, String tenantId){
 
         PropertyCriteria criteria = PropertyCriteria.builder().tenantId(tenantId).uuids(Collections.singleton(id)).build();
 
-        List<Property> properties = propertyRepository.getProperties(criteria, false);
```
Comment (talele08) [inline property-services/src/main/java/org/egov/pt/consumer/FileStoreConsumer.java:100]: Why is this needed?

Comment (rohit-eGov) [reply property-services/src/main/java/org/egov/pt/consumer/FileStoreConsumer.java:100]: In this getProperties() method three parameter need to pass but only two are passing
that's why added third parameter in this method call

**Review by talele08 — APPROVED** [review]
